tests/testthat/test-step-mutate.R

```diff
@@ -105,3 +105,27 @@ test_that("emtpy mutate returns input", {
   expect_equal(mutate(dt), dt)
   expect_equal(mutate(dt, !!!list()), dt)
 })
+
+# .before and .after -----------------------------------------------------------
+
+test_that("can use .before and .after to control column position", {
+  dt <- lazy_dt(data.frame(x = 1, y = 2))
+  expect_named(
+    mutate(dt, z = 1) %>% as_tibble(),
+    c("x", "y", "z")
+  )
+  expect_named(
+    mutate(dt, z = 1, .before = x) %>% as_tibble(),
+    c("z", "x", "y")
+    )
```

Comment:
Indenting is slightly off here and below.

Reply:
Thanks, fixed.